app/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_status_validation_mixin.py

```diff
@@ -56,3 +58,308 @@ def test_update_experiment_with_invalid_publish_status_error(self):
         )
         self.assertFalse(serializer.is_valid())
         self.assertIn("experiment", serializer.errors)
+
+    @parameterized.expand(
+        [
+            [
+                NimbusExperimentFactory.Lifecycles.CREATED,
+                NimbusExperiment.PublishStatus.IDLE,
+                True,
+            ],
+            [
+                NimbusExperimentFactory.Lifecycles.CREATED,
+                NimbusExperiment.PublishStatus.DIRTY,
+                True,
+            ],
+            [
+                NimbusExperimentFactory.Lifecycles.CREATED,
+                NimbusExperiment.PublishStatus.REVIEW,
+                True,
+            ],
+            [
+                NimbusExperimentFactory.Lifecycles.CREATED,
+                NimbusExperiment.PublishStatus.APPROVED,
+                False,
+            ],
+            [
+                NimbusExperimentFactory.Lifecycles.CREATED,
+                NimbusExperiment.PublishStatus.WAITING,
+                False,
+            ],
+        ]
+    )
+    def test_update_publish_status_errors_for_create(self, lifecycle, status, valid):
```

Comment:
I would call this parameter `publish_status` just to make it really clear it's not a `status` which is a different thing.  When I added publish_status I wasn't sure if I should rename `status` to something that also has a qualifier like `lifecycle_status` or something, I just left it alone cuz we'd have to refactor basically the whole app.  But yeah generally I try to name parameters like this after the field they're referring to on the model.